android/app/build.gradle.kts

```diff
@@ -105,7 +112,7 @@ sentry {
   org.set("emerge-tools")
   projectName.set("hackernews-android")
 
-  ignoredVariants.set(listOf("debug", "fast"))
+  ignoredVariants.set(listOf("debug", "fast", "release"))
```

Comment:
Hmm, I think we want to keep release enabled for Sentry

Reply:
that's for my local testing, I keep forgetting to put it back, fixed!